### PR TITLE
reexported glam

### LINKED ISSUE #1067: Export glam
Fixes version mismatches
Resolves the issue of users copying code from examples and wondering why it doesn't compile

### PR DESCRIPTION
**Pull request by PSteinhaus**
what it says in the title
Closes #1067 

Still opened a PR for it, though it's only a single line, so we can directly decide over it here.

### CODE REVIEW

**Comment by nobbele** [comment]
Should update examples to use this export

**Comment by PSteinhaus** [comment]
Good point. Will do.

File: resources/dimmer.wgsl [hunk 1]
```diff
@@ -11,10 +11,10 @@ struct Dim {
 [[group(1), binding(0)]]
 var t: texture_2d<f32>;
 
-[[group(1), binding(1)]]
+[[group(2), binding(0)]]
 var s: sampler;
 
-[[group(3), binding(0)]]
```
Comment (nobbele) [inline resources/dimmer.wgsl:17]: Why did this change? Was it broken before

**Comment by PSteinhaus** [comment]
It was. You can check the current example. It doesn’t run.

**Pull request merged**